data/remnant/remnant 1 introduction.txt

```diff
@@ -1269,13 +1269,13 @@ mission "Remnant: Deep Surveillance"
 			
 			`	"Oh, yes. We haven't been able to do too much with what funding we have left, but we have made one important discovery." Ivan turns to one of his filing cabinets and pulls out some sort of graphic that you would guess is depicting the anomaly, but you can't make any sense of it.`
 			`	"One of our ideas has been that the anomaly is some sort of unstable wormhole. Reports of its existence go back to at least the times of the first Alpha War, with rumors that ships have left for Terminus to never return, presumably having entered the anomaly somehow. Now I'm not one to go off of rumors, but they certainly have to come from somewhere, so the idea was at least worth investigating.`
-			`	"Going under the assumption that it is a wormhole that is simply too unstable to allow a ship to pass through, we wanted to see if we could get anything at all to pass through. What we've discovered is that given a small and smooth enough object, we can get it to disappear within the anomaly without a trace." He then pulls out a small metallic sphere about the size of a ping pong ball from one of the cabinets which he holds onto by a wire.`
+			`	"Going under the assumption that it is a wormhole that is simply too unstable to allow a ship to pass through, we wanted to see if we could get anything at all to pass through it. What we've discovered is that given a small and smooth enough object, we can get it to disappear within the anomaly without a trace." He then pulls out a small metallic sphere about the size of a ping pong ball from one of the cabinets and dangles it from an attached wire.`
 			`	"Now this is only a model, but we created a number of small sensors about this size and sent them through the anomaly. We'd hoped that we could take some readings from whatever is on the other side of the anomaly, as it clearly goes somewhere, but we only ever get static coming back through the wire, followed by nothing a few minutes later. And unfortunately, whatever we send in we can't seem to pull back. The wire always snaps.`
-			`	"So the anomaly is clearly a wormhole. Now we just need to figure out how to get whatever we send into it to come back, and how to get something the size of a ship to enter it."`
+			`	"So the anomaly is clearly a wormhole. Now we just need to figure out how to get whatever we send into it to come back and how to get something the size of a ship to enter it."`
```

Comment:
```suggestion
			`	"So the anomaly is clearly a wormhole. Now we just need to figure out how to get whatever we send into it to come back, and how to get something the size of a ship to enter it."`
```

Reply:
Losing this comma is correct here, I believe. Just a complex predicate.